plugins/ern_v0.14.0+/rn-fetch-blob_v0.12.0+/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": []
```

Comment:
permissions

```
+   <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+   <uses-permission android:name="android.permission.DOWNLOAD_WITHOUT_NOTIFICATION" />
```

Reply:
Added a README for this as these permissions needed to be added based on what features they are going to use.